app/controllers/subscriptions_controller.rb

```diff
@@ -11,15 +13,17 @@ def create
     subscription.frequency = frequency
     subscription.signon_user_uid = current_user.uid
     subscription.save!
-
     render json: { id: subscription.id }, status: status
   end
 
 private
 
+  def notify_address
```

Comment:
To follow ruby convention this method should end with a question mark

Reply:
Also might be better to call this smoke_test_address? be good to have clues that this is just for testing purposes.